modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,20 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+The amount of resources required by Network Observability depends on the size of your cluster and your requirements for ingesting and storing observability data. There are settings to consider that can help you get an optimal setup and observability.
+
+The following settings can help you manage resources and performance from the outset:
+   
+    eBPF Sampling:: You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Lower sampling values might consume a large amount of computational, memory and storage resources. You can mitigate this by specifying a sampling ratio value. A value of 100 means 1 flow every 100 is sampled. A value of 0 or 1 means all flows are captured. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, so 1 flow every 50 is sampled. Note that more sampled flows also means more storage needed. It is recommend to start with default values and refine empirically, to determine which setting your cluster can manage.
+
+    Restricting or excluding interfaces::  Reduce the overall observed traffic using `spec.agent.ebpf.interfaces` and `spec.agent.ebpf.excludeInterfaces`. By default, the agent fetches all the interfaces in the system, except the ones listed in `excludeInterfaces` and 'lo' (local interface). Note that the interface names might vary according to the CNI used.
+
+The following settings can be used to fine-tune performance after Network Observability has been running for a while:
```

Comment:
```suggestion
After Network Observability has stablized for your cluster, you can fine-tune your cluster's performance by configuring the following settings:
```

Reply:
s/After Network Observability/After the Network Observability

Reply:
@dfitzmau 
I'm not sure where you got the _stablized_ (s/b stabilized?). It makes it sound like the cluster was unstable, which is not the case, as I read things. I do like the change to the active voice.
@skrthomas It might be worth a quick question to the dev/qe folks about how to determine when the user should fine-tune.